File: module-gan/src/main/java/com/huyingbao/module/gan/ui/random/adapter/CategoryAdapter.java
package com.huyingbao.module.gan.ui.random.adapter;

import androidx.annotation.Nullable;

import com.chad.library.adapter.base.BaseQuickAdapter;
import com.chad.library.adapter.base.BaseViewHolder;
import com.huyingbao.module.gan.R;

import java.util.List;

/**
 * Created by liujunfeng on 2019/1/1.
 */
public class CategoryAdapter extends BaseQuickAdapter<String, BaseViewHolder> {
    public CategoryAdapter(@Nullable List<String> data) {
        super(R.layout.gan_recycle_item_category, data);
    }

    @Override
    protected void convert(BaseViewHolder helper, String item) {
        helper.setText(R.id.tv_product_name, item);
    }
}

File: module_gan_kotlin/src/main/java/com/huyingbao/module/gan/ui/random/adapter/CategoryAdapter.kt
package com.huyingbao.module.gan.ui.random.adapter

import com.chad.library.adapter.base.BaseQuickAdapter
import com.chad.library.adapter.base.BaseViewHolder
import com.huyingbao.module.gan.R

/**
 * Created by liujunfeng on 2019/1/1.
 */
class CategoryAdapter(data: List<String>?) : BaseQuickAdapter<String, BaseViewHolder>(R.layout.gan_recycle_item_product, data) {

    override fun convert(helper: BaseViewHolder, item: String) {
        helper.setText(R.id.tv_product_name, item)
    }
}
